node_modules/oae-admin/manageuser/js/manageuser.js

```diff
@@ -156,15 +160,86 @@ define(['jquery', 'oae.core'], function($, oae) {
         };
 
         /**
+         * Delete a user
+         */
+        var deleteUser = function() {
+            // Lock the modal so it cannot be closed while deleting the user
+            $('#manageuser-modal', $rootel).modal('lock');
+            $('#manageuser-actions-form .modal-footer .manageuser-actions-deleteuser button').prop('disabled', true);
+
+            oae.api.admin.deleteUser(widgetData.userProfile.id, function(err) {
+                // When the delete request finishes re-enable the form
+                $('#manageuser-modal', $rootel).modal('unlock');
+                $('#manageuser-actions-form .modal-footer .manageuser-actions-deleteuser button').prop('disabled', false);
+
+                if (err) {
+                    // Show an error notification
+                    oae.api.util.notification(
+                        oae.api.i18n.translate('__MSG__USER_NOT_DELETED__', 'manageuser'),
+                        oae.api.i18n.translate('__MSG__COULD_NOT_DELETE_USER_NAME__', 'manageuser', {
+                            'displayName': oae.api.util.security().encodeForHTML(widgetData.userProfile.displayName)
+                        }),
+                        'error');
+                } else {
+                    // Success, reload the page
+                    document.location.reload()
```

Comment:
Infinite scroll has a `removeItems` function that allows you to remove items from an infinite scroll list by id. Rather than reloading the page, I think using that and hiding the modal would be a better experience.